style.css

```diff
@@ -0,0 +1,3273 @@
+@media screen{
+    body {
+        margin: 0;
+        padding: 0;
+        overflow-x: hidden;
+    }
+    
+    h1, h2, h3, h4, h5, p, ul, li{
+        margin: 0;
+        padding: 0;
+    }
+    
+    .css-1fe7a5q {
+        width: 18px;
+        padding-right: 13px;
+    }
+    
+    .css-1fe7a5q2 {
+        width: 18px;
+        padding-right: 13px;
+    }
+    
+    img{
+        max-width: 100%;
+        height: auto;
+    }
+    main {
+        max-width: 1200px;
+        margin:0 auto;
+        margin-top: 5px;
+        padding: 0 1% ;
+    }
+    
+    .top_nav {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+    
+    .top_nav-second ul {
+        display: flex;
+        list-style-type: none;
+        justify-content: center;
+        width: 120%;
+        margin-left: 58px;
+    
+    }
+    
+    
+    
+    .top_nav-second ul li {
+        margin-right: 15px;
+    }
+
+    
+    
+    .top_nav-second a{
+        text-decoration: none;
+        color: black;
+        font-size: 10px;
+        font-weight: bold;
+    }
+    
+    .top_nav-second a:hover{
+        background-color: rgb(231, 224, 224);
+        color: black;
+        padding: 7px 8px 6px;
+    }
+    
+    
+    
+    
+    .top_nav-third button{
+        background-color: #567B95;
+        color: rgb(244, 243, 243);
+        /* font-size: 80%;
+        font-weight: 500; */
+        padding: 8px 12px;
+        border: #567B95;
+        border-radius: 2px;
+        font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    }
+    
+    /* END OF TOP NAV */
+    
+    
+    
+    .highlight{
+        align-items: center;
+        margin-left: 20%;
+    }
+    
+    .highlight-list{
+        display: flex;
+        list-style-type: none;
+        font-size: 90%;
+    }
+    
+    .live{
```

Comment:
if you are using padding in general then u shouldn't use padding-top or bottom after that. Only the last rule applies so whatever you write before becomes useless.